### PR TITLE
chore: Update issue templates

### PR DESCRIPTION
**Pull request by ganeshnj**
Included bug report and feature request templates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+name: Bug report
+about: Create a report to help us improve
```
Comment (maciejburda) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]:
```suggestion
about: Create a report to help us improve the SDK.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+The code sample should be an SSCCE. See http://sscce.org/ for details. In short, please provide a code sample that we can copy/paste, run and reproduce.
+
+### Expected behavior
+A clear and concise description of what you expected to happen.
+
+---
+
+#### Datadog SDK version:
+
+_Which version of the Datadog SDK causes this problem? e.g. `1.2.0`_
```
Comment (maciejburda) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: maybe worth setting `2` as a major version for both examples (one less char to edit)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+### Describe the bug
+A clear and concise description of what the bug is.
+
+### Reproduction
+
+Provide a self-contained, concise snippet of code that can be used to reproduce the issue.
+For more complex issues provide a repo with the smallest sample that reproduces the bug.
+
+Avoid including business logic or unrelated code, it makes diagnosis more difficult.
```
Comment (maciejburda) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: Somewhere along these lines, worth mentioning that they should redact any secrets. It's easy to copy paste tokens when providing examples.

**Review by maciejburda — COMMENTED** [review]
Good stuff. Added some minor comments.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (ganeshnj) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: for sure.

Updated.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (ganeshnj) [reply .github/ISSUE_TEMPLATE/bug_report.md:28]: done

**Review by ncreated — APPROVED** [review]
👍

**Pull request merged**